Sources/MongoSwift/MongoClient.swift

```diff
@@ -278,6 +278,96 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+    /**
+     * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+     * system collections i.g. "config", "local", and "admin" databases. By default, the type `CollectionType` is
+     * associated with the `fullDocument` field in the `ChangeStreamsDocument`.
+     * - Parameters:
+     *   - Pipeline: The pipeline of stages to append to an initial `ChangeStream` stage.
```

Comment:
1. "Pipeline" -> "pipeline" so it matches the capitalization of the parameter
I think mentioning the "initial changestream stage" is a little confusing because users might not necessarily realize this helper is implemented via aggregate. 
it might be clearer to say something like "An array of aggregation pipeline stages to apply to the events returned by the change stream." 

same applies to other places this parameter is used